docs/team/tengjianling.md

```diff
@@ -1,52 +1,52 @@
-- Changed restrictions on `question`, `answer`, and `tag` fields in `add` command: [#66](https://github.com/AY2021S1-CS2103T-W15-1/tp/pull/66)
-- Changed implementation of `QuizCommand` and `QuizCommandParser` to include ability to quiz based on multiple keywords or tags: [#73](https://github.com/AY2021S1-CS2103T-W15-1/tp/pull/73)
-- Wrote the following additional tests: `QuizCommandTest`, `QuizCommandParserTest`, `ExitCommandTest` and `ExitCommandParserTest`: [#73](https://github.com/AY2021S1-CS2103T-W15-1/tp/pull/73)
-  - increased code coverage by 2.96%
-- Updated the GUI for quiz to be centralised and remove index number: [#83](https://github.com/AY2021S1-CS2103T-W15-1/tp/pull/83)
-
-
-
-### Documentation:
-
-- User Guide:
-  - Added documentation for the features `add` , `delete`, and `edit`: [#18](https://github.com/AY2021S1-CS2103T-W15-1/tp/pull/18), [#83](https://github.com/AY2021S1-CS2103T-W15-1/tp/pull/83)
-- Developer Guide:
-  - Added use cases for Medmoriser: [#18](https://github.com/AY2021S1-CS2103T-W15-1/tp/pull/18)
-  - Added implementation details of the `delete` feature, including a sequence diagram and activity diagram: [#78](https://github.com/AY2021S1-CS2103T-W15-1/tp/pull/78)
-
-### Community:
-
-- PRs reviewed (with non-trivial review comments): [#71](https://github.com/AY2021S1-CS2103T-W15-1/tp/pull/71)
-- Reported bugs and suggestions for other teams in the class (examples: [1](https://github.com/AY2021S1-CS2103T-W16-3/tp/issues/236), [2](https://github.com/AY2021S1-CS2103T-W16-3/tp/issues/237))
+- **New feature:** Added the feature to allow user to be quized on the next question in the quiz.
+  - What it does: allows you to be tested on another question based on the same keywords you provided for the quiz, without having to end the quiz first.
+  - Justification: It may be troublesome for the user to have to go through the process of ending the quiz and typing the whole quiz command again if he wants to quiz himself with the same keywords, and he may get the same exact same question as before even if it is randomised. This `next` command provides a convenient way for users to continually quiz themselves.
+  - Highlights: The implementation of this enhancement was challenging as additional fields have to be created to keep track of which questions to randomise from, since we do not want the `qAndA` obtained from the `next` command to be the same as the current `qAndA` that the user is seeing.
+
+- **Enhancements to Existing features:**
+  - Changed the usage message in `edit` command to the context of questions and answers: [#41](https://github.com/AY2021S1-CS2103T-W15-1/tp/pull/41)
+  - Renamed all uses of `QuestionSet` and `questionSet` to `QAndA` and `qAndA` to avoid confusion between sets and lists: [#43](https://github.com/AY2021S1-CS2103T-W15-1/tp/pull/43)
```

Comment:
same here

Reply:
I meant renaming instance names to qAndA here

Reply:
sorry did not see that.